tests/test_samplers.py

```diff
@@ -11,7 +11,7 @@
 
 from pykeen.datasets import Nations
 from pykeen.sampling import BasicNegativeSampler, BernoulliNegativeSampler, NegativeSampler
-from pykeen.sampling.pseudo_type_negative_sampling import PseudoTypedNegativeSampler
+from pykeen.sampling.pseudo_type import PseudoTypedNegativeSampler
```

Comment:
import this through the `from pykeen.sampling` the same as the basic and bernoulli ones

Reply:
3a5729b